test/grep.js

```diff
@@ -127,6 +134,15 @@ test('one file, * in string-regex, make sure * is not globbed', t => {
   t.is(result.toString(), 'this line ends in.js\nlllllllllllllllll.js\n');
 });
 
+test("one file, pattern doesn't match", t => {
+  const result = shell.grep('notfoundstring', 'test/resources/grep/file');
+  t.truthy(shell.error());
+  t.is(result.toString(), '');
+  t.is(result.stdout, '');
+  t.is(result.stderr, 'grep: ');
```

Comment:
```suggestion
  // TODO(#900): "grep: " isn't really the correct stderr output, but we need a
  // non-empty string so `shell.error()` is truthy.
  t.is(result.stderr, 'grep: ');
```

Reply:
Also, please double-check that the suggestion's line-length falls within our limit